core/src/executor/insert.rs

```diff
@@ -40,29 +40,29 @@ pub enum InsertError {
     MapTypeValueRequired(String),
 }
 
-enum RowsData {
+pub enum RowsData {
     Append(Vec<DataRow>),
     Insert(Vec<(Key, DataRow)>),
 }
 
+pub async fn fetch_insert_rows<T: GStore + GStoreMut>(
+    storage: &mut T,
+    table_name: Option<&str>,
+    columns: &[String],
+    query: &Query,
+    column_defs: Option<&Vec<ColumnDef>>,
```

Comment:
We can use `column_defs: Option<&[ColumnDef]>` in here.
Then, we can also replace `as_ref` to `as_deref` for the codes which use this.